pixart.js

```diff
@@ -1 +1,33 @@
+var brush = $('.brush')
+var button = $('#set-color')
+var input = $('#color-field')
+var body = $('body')
+// brush.css('background-color', '#1B4370')
+// $('brush').css('background-color', 'blue');
 
+var setColor = function (e) {
+  e.preventDefault()
+  brush.css('background-color', input.val())
+}
+
+button.on('click', setColor)
+
+for (i = 0; i < 8000; i++) {
+var square = $('<div class ="square"></div>')
```

Comment:
Watch your indentation, This var should be moved in to match the indentation of the next lines